SEM-Party-Directory.md

```diff
@@ -0,0 +1,24 @@
+# SEM party directory
+
+The following parties are available in the SEM eco system. This list acts a s a temporary directory for all endpoints and contact persons.
+
+Please contact the contact person to exchange authentication information like authentication server url,client-id and secrets. 
+
+Please contact stichting SEM to be included in this directory.
+
+## Region Netherlands
+
+|  Name | Product | Role | Endpoint URl | Contatc info |
```

Comment:
Authentication provider URL / Scope that needs to be send

Reply:
I will add the URL, the scopes are part of the api spec